samples/basic/minimal/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _minimal:
```

Comment:
Because these doc labels must be unique across all the documentation, a better name would be ``minimal_sample``